templates/Web/_composition/LaunchScript/Launch.ReactNode.AddScript/package$Launch.ReactNode.AddScript_gpostaction.json

```diff
@@ -0,0 +1,5 @@
+  "scripts": {
+//{[{
+    "start": "concurrently \"react-scripts start\" \"nodemon server/server.js\"",
+//}]}
+  },
```

Comment:
It seems like this line ("start": "concurrently....) will get inserted into the **scripts** object inside **package.json** of generated templates.

Just two learning questions, 1) how does coreWebTemplate know which file it should insert code into and 2) which file it should grab the line it is going to use for insertion?

Reply:
it knows due to the file name notice "package_postaction.json", or in this case "package$<some id>_gpostaction.json". The difference between the two is that package_postaction will be merged after the page/project template is merged but gpostaction will be merged after all other _postactions and templates are merged. So they are like a final merging step, this is especially helpful for guarantee of a certain format/existence of a certain file that may be added in a _postaction.

Reply:
To my knowledge it searches for the `_postaction/$<id>_gpostaction` tags, and then it determines the original file it needs to insert into by removing the tag, so for our case it would be "package.json". Does this make sense?